src/_locales/ja/messages.json

```diff
@@ -0,0 +1,358 @@
+{
+    "about": "概要",
+    "about_dialog_description": "Maskbookは大企業からのビッグデータ監視から逃れるためにSNSへの投稿を暗号化できます。あなたと友だちだけが暗号化された投稿を解読できます。",
+    "about_dialog_touch": "連絡する",
+    "about_dialog_feedback": "フィードバック: ",
+    "about_dialog_source_code": "ソースコード: ",
+    "about_dialog_license": "ライセンス: ",
+    "add_persona": "人格を登録する",
```

Comment:
I know that 人格 is not good choice, but 身分 is also (It may means like position or status). To tell the truth, I think there are no best Japanese word for persona...

Reply:
I searched some dictionaries and found no common word that shows best meaning.
I used "人格" in the meaning of the multiple face inside the human like 裏垢(sub accounts).
I suggest to leave as it is until the best word had risen up in someone's mind.

Reply:
Thanks!!